Makefile

```diff
@@ -15,7 +15,7 @@ clients: linux-client macos-client windows-client
 # Before building linux-client and debian package, these are needed
 linux-init:
 	sudo apt update
-	sudo apt install build-essential libgtk-3-dev libappindicator3-dev ruby ruby-dev rubygems
+	sudo apt install build-essential libgtk-3-dev libappindicator3-dev ruby ruby-dev rubygems imagemagick
```

Comment:
This is not necessary - the deb installation works fine for me without. So again, maybe describe _why_ this is a worthwhile addition (in eg. the commit message) instead of just stating that you added it?

Reply:
It's required to make the deb, and not present on all systems (i.e. ubuntu-minimal)

Reply:
If you're certain of that, I'll trust you on it! It'd still be nice with explicit rationale for _why_ instead of just a repetition of _what_ in git commit messages though =)

Reply:
When generating icons for linux, we use `convert` (from imagemagick) to resize them. You might have imagemagick (or a compatability package) installed already? Or there is something wrong with the dependencies in the Makefile so it never generated the icons on your machine... (truth be told, there are a lot of missing dependency declarations in the Makefile, so this would not be surprising. I did try to fix those that popped up while working on this, but I might have missed some).

Reply:
Ok